### PR TITLE
Update building-blocks.md

### PR DESCRIPTION
**Pull request by huyndo**
Fix broken links

References: https://github.com/onflow/developer-portal/issues/735

______

For contributor use:

- [x] Targeted PR against `master` branch
- [x] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [x] Updated relevant documentation 
- [x] Re-reviewed `Files changed` in the Github PR explorer
- [x] Added appropriate labels

### CODE REVIEW

File: docs/content/nft-marketplace/building-blocks.md [hunk 1]
```diff
@@ -22,14 +22,14 @@ If you are coming from Ethereum, the following list shows corresponding modules
...
-| Stablecoins                         | USDC/DAI/USDT           | [FUSD](fusd/), tUSDT, USDC                                                                            |
-| Wallet Interaction Library          | web3.js                 | [Flow Client Library (FCL)](fcl/)                                                                     |
-| Blockchain Interaction SDK          | web3.js                 | [Flow Client Library (FCL)](fcl/), [Flow Go SDK](flow-go-sdk/), and [many others](flip-fest-winners/) |
+| Native Crypto Currency              | ETH                     | [FLOW](https://docs.onflow.org/flow-token/)                                                                                   |
+| Stablecoins                         | USDC/DAI/USDT           | [FUSD](https://docs.onflow.org/fusd/), tUSDT, USDC                                                                            |
+| Wallet Interaction Library          | web3.js                 | [Flow Client Library (FCL)](https://github.com/onflow/fcl-js)                                                                     |
```
Comment (bthaile) [inline docs/content/nft-marketplace/building-blocks.md:31]: `fcl/` should be fine as well. it gets redirected, should only change the broken links.
fcl/
flow-go-sdk//

also, use "https://developers.flow.com" as the root cuz docs.onflow.org gets redirected

File: docs/content/nft-marketplace/building-blocks.md [hunk 2]
```diff
@@ -22,14 +22,14 @@ If you are coming from Ethereum, the following list shows corresponding modules
 
 | Resource                            | Ethereum                | Flow                                                                                                  |
 | ----------------------------------- | ----------------------- | ----------------------------------------------------------------------------------------------------- |
-| Smart Contract Programming Language | Solidity                | [Cadence](cadence/)                                                                                   |
```
[the remark this answers is not in the document]
Comment (huyndo) [reply docs/content/nft-marketplace/building-blocks.md:25]: It goes into https://developers.flow.com/flow/nft-marketplace/cadence instead unfortunately.

File: docs/content/nft-marketplace/building-blocks.md (hunk 1 above)
Comment (huyndo) [reply docs/content/nft-marketplace/building-blocks.md:31]: They still resolve to `flow/nft-marketplace/fcl/` or `flow-go-sdk/`. Is there a place I can change for the redirects?

Comment (bthaile) [reply docs/content/nft-marketplace/building-blocks.md:31]: links that don't start with '/' will get prepended with "https://developers.flow.com/"
so `(FCL)](fcl/) ` will resolve to https://developers.flow.com/fcl which does get redirected to tools/fcl-js
but they are not broken so shouldn't be changed in this PR.

Comment (huyndo) [reply docs/content/nft-marketplace/building-blocks.md:31]: Is this resolve currently live? I'm just basing off what's happening on https://developers.flow.com/flow/nft-marketplace/building-blocks where the links are not redirected properly.

**Review by bthaile — APPROVED** [review]
good to go, fixes broken links. thx

**Pull request merged**